sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -280,6 +279,14 @@ export class ReceiveHandler {
     stop(): Promise<void>;
 }
 
+// @public
+export interface ReceiverRuntimeInfo {
```

Comment:
The name should be `LastEnqueuedEventInfo`
The properties on it can be
- offset
- sequenceNumber
- enqueuedTime
- retrievalTime